addons/sys_core/fnc_processRadioSpeaker.sqf

```diff
@@ -43,6 +50,12 @@ if (!GVAR(speaking_cache_valid)) then {
     #endif
     _okRadios = (_okRadios select 0) select 1;
 
+    // Debug #638
+    if (isNil "_radioId") exitWith {
```

Comment:
Should this nil check be on `_okRadios`? _radioId can't change from the last check
also exitWith return value will be lost to scope

Reply:
That's what I am trying to figure out, somehow `_radioId` is `nil` in the below `HASH_SET` operation, even though it should be `""` due to `getVariable` default value on `nil`.

Does `setVariable` also throw "reserved variable" error if you try to assign nil to it? I don't think so.